doc/api/test.md

```diff
@@ -1219,6 +1239,42 @@ A successful call to [`run()`][] method will return a new {TestsStream}
 object, streaming a series of events representing the execution of the tests.
 `TestsStream` will emit events, in the order of the tests definition
 
+### Event: `'test:coverage'`
+
+* `data` {Object}
```

Comment:
Will this output be compatible with something like lcov (or another format tools are used to read)?

Reply:
No. The test reporter interface is used to get information out of the test runner. You would need to write an lcov reporter.